webpack.config.js

```diff
@@ -111,6 +118,42 @@ var wpconfig = {
                 use: 'babel-loader'
             },
             {
+                test: /\.(js|jsx)$/,
+                include: [
+                    /studio-frontend/,
+                    /paragon/
+                ],
+                use: 'babel-loader'
+            },
+            {
+                test: /(.scss|.css)$/,
+                include: [
+                    /studio-frontend/,
+                    /paragon/,
+                    /font-awesome/
+                ],
+                use: extractSass.extract({
+                    use: [{
+                        loader: 'css-loader',
+                        options: {
+                            modules: true,
+                            localIdentName: '[name]__[local]___[hash:base64:5]'
+                        }
+                    }, {
+                        loader: 'sass-loader',
+                        options: {
+                            data: '$base-rem-size: 0.625; @import "paragon-reset";',
```

Comment:
Is `$base-rem-size` updated to keep the font from being tiny? Is this a bootstrap thing?

Reply:
I believe so. @arizzitano, can you confirm? I remember adding this when I copied one of your sass-loader configs, but I can't seem to find where I copied it from...